src/main/java/org/gridsuite/network/map/dto/line/LineTooltipInfos.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.network.map.dto.line;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.powsybl.iidm.network.Identifiable;
+import com.powsybl.iidm.network.Line;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.extensions.BranchStatus;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.network.map.model.CurrentLimitsData;
+
+import static org.gridsuite.network.map.dto.utils.ElementUtils.nullIfNan;
+import static org.gridsuite.network.map.dto.utils.ElementUtils.toMapDataCurrentLimits;
+
+/**
+ * @author Le Saulnier Kevin <kevin.lesaulnier at rte-france.com>
+ */
+@SuperBuilder
+@Getter
+public class LineTooltipInfos extends AbstractLineInfos {
+
+    private String voltageLevelId1;
+
+    private String voltageLevelId2;
+
+    private Boolean terminal1Connected;
+
+    private Boolean terminal2Connected;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
You don't need p1 and p2 nope ?

Reply:
fixed

Reply:
removed in TwoWindingsTransformerTooltipInfos.java as well